javascript_intro/events.js

```diff
@@ -1,16 +1,62 @@
-let add_item_to_list = (dom_element_creator, dom_element_adder) => {
+//Array de Kpi_measures
+let measures = new Array();
+//Función de Average
+function average(arr){
+  let sum = arr.reduce((acc, cur) => acc + cur, 0)
+  let answer = sum / arr.length
+  return answer
+}
+//Función de Total
```

Comment:
yes you can use reduce, but don't use a, b as var names, use proper names that specify what they actually are